### DISCUSSION TITLE
App Request

### DISCUSSION THREAD

**Discussion by lucky-yoolk** [dr-slash/icons-flat-osx#101] [Requests]
Add your request in the comments below ;)

## APPS
- [ ] Amazon Drive
- [ ] Amazon /Photos
- [ ] App Delete
- [ ] Audio Hijack
- [ ] Aurora HDR
- [ ] Bartender
- [ ] [Bee](http://www.neat.io/bee/index.html)
- [ ] Bitcoin-Qt
- [ ] BlackBerry
- [ ] Candybar
- [ ] cDock
- [ ] Chess
- [ ] Clearview
- [ ] Clipgrab
- [ ] Command-Tab Plus
- [ ] Crossover
- [ ] Datagrip
- [ ] DaVinci Resolve
- [ ] Deliveries
- [ ] Digital Color Meter
- [ ] Disk Inventory X
- [ ] Dolphin Emulator
- [ ] DVDFab
- [ ] EasyFind
- [ ] Emacs
- [ ] Fantastical
- [ ] Finale 2014
- [ ] Flamerobin
- [ ] Flavours 2
- [ ] Flickery
- [ ] Font Lab
- [ ] FruitJuice
- [ ] GameJolt Client
- [ ] GeekBench 3
- [ ] GeForce Experience
- [ ] Get Plain Text
- [ ] [Godot Engine](https://godotengine.org/)
- [ ] Google Keep
- [ ] GPG keychain
- [ ] IINA
- [ ] Incognito
- [ ] Intensify
- [ ] iStat Menus
- [ ] iVI
- [ ] JPEGmini Pro
- [ ] Keychain Access
- [ ] Kindle
- [ ] LabVIEW
- [ ] LastPass
- [ ] [Launchpad Manager](http://launchpadmanager.com/)
- [ ] Logitech Options
- [ ] LTSpice
- [ ] MacGameStore
- [ ] Macs Fan Control
- [ ] MatLab 2014
- [ ] Media Inspector
- [ ] Microsoft Access
- [ ] Microsoft Publisher
- [ ] Migration Assistant
- [ ] MKVToolNix
- [ ] Movist
- [ ] [mSecure](https://www.msecure.com/)
- [ ] MullvadVPN
- [ ] OmniDiskSweeper
- [ ] OmniGraffle
- [ ] [Path Finder](http://www.cocoatech.com/pathfinder/)
- [ ] pgAdmin
- [ ] PlistEditPro
- [ ] Python Folder & Python IDE
- [ ] Qbittorrent
- [ ] Qobuz
- [ ] Quicksilver
- [ ] R
- [ ] Reaper
- [ ] Right Font
- [ ] RStudio
- [ ] SCP
- [ ] Script Editor
- [ ] SiteSucker
- [ ] Skitch
- [ ] Spacemacs
- [ ] SQL Lite Browser
- [ ] SQLDeveloper
- [ ] Tidal
- [ ] TinkerTool
- [ ] Titanium Studio
- [ ] Tunnelblick
- [ ] Ulysses
- [ ] Unetbootin
- [ ] Unibox
- [ ] Uplay
- [ ] Vienna
- [ ] VNC Server
- [ ] VoiceOver Utility
- [ ] [Web Torrent](https://webtorrent.io/)
- [ ] Winzip
- [ ] Wolfram Mathematica 10
- [ ] XLD
- [ ] Yate
- [ ] Yoink
## GAMES & EMULATORS
- [ ] 140
- [ ] Andy Emulator
- [ ] Arma 3
- [ ] Bastion
- [ ] BlueStack Emulator
- [ ] Borderlands 2
- [ ] Cities: Skylines
- [ ] Dirt 3
- [ ] Dolphin
- [ ] Don't Starve / Don't Starve Together
- [ ] Doom Emacs (A Cacodemon from doom)
- [ ] Dota 2
- [ ] FIFA 18
- [ ] Garry's Mod
- [ ] Grid 2
- [ ] PCSX2
- [ ] PCSXR
- [ ] Rust
- [ ] [Sleeping Dogs: Definitive Edition ](http://store.steampowered.com/app/307690/)
- [ ] Team Fortress 2
- [ ] [Thief](http://www.thiefgame.com/agegate?locale=us) 
- [ ] This war of mine
- [ ] Universal Emulator (win)
- [ ] VisualBoyAdvance
## FOLDERS
- [ ] Toast Titanium
- [ ] MS Office
- [ ] Mac Game Store
## STARTUP VOLUMES
- [ ] OSX
- [ ] Windows
- [ ] Linux

**Comment by ghost** [comment]
please make a FLAT OSX ICON for *GENSHIN IMPACT*

**Reply by lucky-yoolk** [reply]
https://synth.agency/icns/genshin-impact/

**Comment by gaetgu** [comment]
Star Craft 1998, INSIDE

**Reply by lucky-yoolk** [reply]
SC app https://synth.agency/icns/stacraft-1-flat-icon/
SC folder https://synth.agency/icns/stacraft-1-folder-flat-icon/

**Reply by gaetgu** [reply]
Thanks so much!!!

**Comment by ghost** [comment]
Wow, never really thought you were actually gonna make that icon. I
genuinely appreciate it 🥺🥺.

**Comment by ghost** [comment]
Any chance for a Rocket League icon?

**Comment by joel-williams** [comment]
RStudio would be so good! It's my one outstanding icon :)

**Comment by PapoBoss** [comment]
Lunar Client icon

**Reply by lucky-yoolk** [reply]
> Lunar Client icon
> 
> 

This app https://lunar.fyi ?

**Comment by dragonstone2** [comment]
Hi! Could you do all of the streaming services? :)

**Comment by cdurkin35** [comment]
Could you make an icon for Executable files?

**Reply by lucky-yoolk** [reply]
Hi, a default .exe file icon?

**Reply by cdurkin35** [reply]
Yes!

**Comment by andrew-turrentine** [comment]
Just discovered and love these icons - thanks for creating and sharing these! My only request would be for JetBrains Rider so I can have an icon for all my JetBrains IDEs.
